spynnaker/pyNN/models/neuron/synaptic_matrices.py

```diff
@@ -546,7 +546,7 @@ def get_connections_from_machine(
 
         :param ~spinnman.transceiver.Transceiver transceiver:
             Used to read the data from the machine
-        :param ~pacman.model.placements.Placements placements:
+        :param ~pacman.model.placements.Placements placement:
```

Comment:
"placements.Placement" not "placements.Placements"

Reply:
fixed